### PR TITLE
class DummyCalibrator replaced by two classes that use a different interpretation of the input values: DummyProbabilityCalibrator and DummyLogOddsCalibrator

### PR DESCRIPTION
**Pull request by wowtor**
class DummyCalibrator replaced by two classes that use a different interpretation of the input values: DummyProbabilityCalibrator and DummyLogOddsCalibrator

### CODE REVIEW

**Comment by vergep** [comment]
Hi Wouter,

I see that the calibration input is either a probability or a log-odds. And you return odds in both cases. So you do not return likelihood ratios, since no base rate correction is applied. Therefore, I find your description of the objects: """
    Calculates a likelihood ratio of a score value, ... """ confusing. This transformer does not calculate LRs, it calculates odds. Question: does LIR also have a base_rate calibrator?

File: lir/calibration.py [hunk 1]
```diff
@@ -632,7 +632,7 @@ def _four_pl_model(s, a, b, c, d):
         return p
 
 
-class DummyCalibrator(BaseEstimator, TransformerMixin):
+class DummyProbabilityCalibrator(BaseEstimator, TransformerMixin):
     """
     Calculates a likelihood ratio of a score value, provided it is from one of
```
Comment (vergep) [inline lir/calibration.py]: likelihood ratio is a confusing term here since you return odds

File: lir/calibration.py [hunk 2]
```diff
@@ -643,10 +643,28 @@ class 1.
...
-        return to_odds(self.p1)
+    def transform(self, probabilities: np.ndarray):
+        self.p0 = (1 - probabilities)
+        self.p1 = probabilities
+        return to_odds(probabilities)
+
+
+class DummyLogOddsCalibrator(BaseEstimator, TransformerMixin):
+    """
+    Calculates a likelihood ratio of a score value, provided it is from one of
```
Comment (vergep) [inline lir/calibration.py]: Same here

File: lir/calibration.py (hunk 1 above)
Comment (wowtor) [reply lir/calibration.py]: Agreed, I updated the docs. There is no base rate calibrator. These classes could be extended for that purpose if there is a need for that.

Comment (vergep) [reply lir/calibration.py]: I think there is a need for that. Scorers are often based on SKlearn classifiers that return (if well calibrated) posterior odds as scores. Then only a 'base rate' calibration is needed to transform them to LRs. I don't know if we want this implemented in this PR however.

Comment (wowtor) [reply lir/calibration.py]: Right, that could be addressed separately.

File: lir/calibration.py [hunk 3]
```diff
@@ -632,21 +632,39 @@ def _four_pl_model(s, a, b, c, d):
...
+        self.p0 = (1 - probabilities)
+        self.p1 = probabilities
+        return to_odds(probabilities)
+
+
+class DummyLogOddsCalibrator(BaseEstimator, TransformerMixin):
+    """
+    Dummy calibrator class which can be used to skip calibration. No
+    calibration is applied. Instead, prior odds of 1 are assumed, and the input
+    values are interpreted as posterior odds. Under these circumstances this
```
Comment (rolfypma-nfi) [inline lir/calibration.py:656]: log odds, apparantly?

**Review by rolfypma-nfi — APPROVED** [review]

**Pull request merged**